modules/logging-loki-storage-aws.adoc

```diff
@@ -0,0 +1,24 @@
+// Module is included in the following assemblies:
+// logging/cluster-logging-loki.adoc
+//
+:_mod-docs-content-type: PROCEDURE
+[id="logging-loki-storage-aws_{context}"]
+= AWS storage
+
+.Prerequisites
+* You have deployed Loki Operator.
+* You have created a link:https://docs.aws.amazon.com/AmazonS3/latest/userguide/create-bucket-overview.html[bucket] on AWS.
+* You have created an link:https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies.html#policies_resource-based[AWS IAM Policy and IAM User].
+
+.Procedure
+* Create an object storage secret with the name `logging-loki-aws` by running the following command:
+
+[source,terminal,subs="+quotes"]
+----
+oc create secret generic logging-loki-aws \
```

Comment:
This is a global merge review comment: all the `oc` commands in this PR need a `$`